functions/rate-limiter/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const FirestoreRateLimiter = require('./firestore-rate-limiter');
+const RedisRateLimiter = require('./redis-rate-limiter');
```

Comment:
There seems to be quite a lot of duplicated paths between these two.
Would it not make sense to have a RateLimiterBase that both of them extend with the different parts?

Reply:
There are some, but making abstract classes in Javascript is a real pain.... once I move it to TypeScript it'll be easier to merge the common bits.